configuration/init/systemd/tedge-mapper.service

```diff
@@ -4,7 +4,7 @@ After=syslog.target network.target mosquitto.service
 
 [Service]
 User=tedge-mapper
-ExecStart=/usr/bin/tedge_mapper
+ExecStart=/usr/bin/tedge_mapper c8y
```

Comment:
I believe this is incorrect as now every time we start mapper via systemd it will start c8y mapper, most likely we want to have separate services invoked with appropriate mapper, isn't it?

Reply:
Yes, it's incorrect. However, if I merge this change, it will break this systemd service. Also, the systemd service for Azure mapper is out of scope in this task. I put `c8y` here as a temporary fix. We will have separated service.

Reply:
Yes, that makes more sense now.